db/migrations/0002_user_order_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.0.2 on 2024-11-12 13:01
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
```

Comment:
Creating a custom `User` model can be useful for extending user functionality, but ensure that this is necessary and that the `AUTH_USER_MODEL` setting in your Django settings is updated to point to this model.